index.js

```diff
@@ -216,6 +215,7 @@ NodeListIterator.prototype.next = function () {
  * @return {Node} The heap's new minimum node.
  */
 function cut(node, parent, minNode, compare) {
+  node.parent = undefined;
```

Comment:
I copied it to https://github.com/gwtw/js-fibonacci-heap/issues/11, feel free to do another PR for the test. If not I'll get to it eventually.